File: app/src/main/java/com/myjb/dev/recyclerView/PriceItemView.java
package com.myjb.dev.recyclerView;

import android.content.Context;
import android.view.LayoutInflater;

import com.myjb.dev.mygaragesale.databinding.ItemRecyclerviewPriceBinding;
import com.myjb.dev.network.PriceItem;

public class PriceItemView extends ItemView {

    ItemRecyclerviewPriceBinding binding;

    public PriceItemView(Context context) {
        super(context);
        binding = ItemRecyclerviewPriceBinding.inflate(LayoutInflater.from(context));
        addView(binding.getRoot());
    }

    @Override
    protected void bind(Object item) {
        if (item instanceof PriceItem) {
            binding.price.setText(((PriceItem) item).price);
            binding.best.setText(((PriceItem) item).best);
            binding.better.setText(((PriceItem) item).better);
            binding.good.setText(((PriceItem) item).good);
        }
    }
}

File: app/src/main/java/com/myjb/dev/recyclerView/PriceItemView.kt
package com.myjb.dev.recyclerView

import androidx.recyclerview.widget.RecyclerView
import com.myjb.dev.mygaragesale.databinding.ItemRecyclerviewPriceBinding
import com.myjb.dev.network.PriceItem

class PriceItemView(private val view: ItemRecyclerviewPriceBinding) :
    RecyclerView.ViewHolder(view.root) {

    fun bind(item: PriceItem) {
        view.price.text = item.price
        view.best.text = item.best
        view.better.text = item.better
        view.good.text = item.good
    }
}